packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -123,7 +123,7 @@ export const GridComponent = React.forwardRef<HTMLDivElement, GridComponentProps
     }
 
     return (
-      <AutoSizer onResize={onResize}>
+      <AutoSizer onResize={onResize} nonce={props.nonce}>
```

Comment:
Should it be on on the `AutoSizer` or `GridRoot` 🤔

Reply:
What do you mean? `nonce` is not an HTML attribute in case the confusion come from there.